assets/html/scss/documenter/components/_footnote.scss

```diff
@@ -0,0 +1,35 @@
+.footnote-reference {
```

Comment:
Maybe name these, e.g., `.hover-reference` and `.hover-preview` to make them more general for things beyond footnotes, like citations?

Reply:
for now until jQuery is being used we can only use this for footnotes but if can we use Julia for adding content then it will be better to change this to global use for all preview features!